app/stock/__init__.py

```diff
@@ -30,13 +30,18 @@ def run_service():
             logger.debug("sleep when market has not opened yet")
             analysis.clear_daily_storage()
             sleep_time = - int(market_info['utcnow - start']) - 60 * 3
+
         if holdings is not None:
+            logger.debug("holding")
+            logger.debug(holdings)
```

Comment:
This can cause a race condition that "holding" being logged to the end of a file and `holdings` being logged to the beginning of a new file.